src/EA.Weee.Web/Services/OrganisationTransactionService.cs

```diff
@@ -59,7 +59,7 @@ public async Task CaptureData<T>(string accessToken, T model) where T : class
                         transaction.AuthorisedRepresentative = authorisedRepresentativeViewModel.SelectedValue.GetValueFromDisplayName<YesNoType>();
                         break;
                     case ContactDetailsViewModel contactDetailsViewModel:
-                        transaction.ContactDetails = contactDetailsViewModel;
+                        transaction.ContactDetailsViewModel = contactDetailsViewModel;
```

Comment:
@will-hope98 Does this require the OrganisationTransactionService unit tests to be updated?

Reply:
Yep added a test to cover that